modules/rgbd/src/large_kinfu.cpp

```diff
@@ -13,6 +13,81 @@ namespace large_kinfu
 {
 using namespace kinfu;
 
+Ptr<VolumeParams> VolumeParams::defaultParams(VolumeType _volumeKind)
+{
+    VolumeParams params;
+    params.kind = _volumeKind;
+    params.maxWeight = 64;
+    params.raycastStepFactor = 0.25f;
+    params.unitResolution = 0;  // unitResolution not used for TSDF
+    float volumeSize = 3.0f;
+    Matx44f pose = Affine3f().translate(Vec3f(-volumeSize / 2.f, -volumeSize / 2.f, 0.5f)).matrix;
+    params.pose = Mat(pose);
+
+    if (params.kind == VolumeType::TSDF)
+    {
+        params.resolutionX = 512;
+        params.resolutionY = 512;
+        params.resolutionZ = 512;
+        params.voxelSize = volumeSize / 512.f;
+        params.depthTruncThreshold = 0.f;  // depthTruncThreshold not required for TSDF
+        params.tsdfTruncDist = 7 * params.voxelSize;  //! About 0.04f in meters
```

Comment:
> `//!`

There is not doxygen in the source code.
Comment is incorrect due to missing `<`.

Reply:
done